addons/binding/pom.xml

```diff
@@ -58,6 +58,8 @@
     <module>org.openhab.binding.toon</module>
     <module>org.openhab.binding.urtsi</module>
     <module>org.openhab.binding.vitotronic</module>
+    <!--module>org.openhab.binding.zwave</module-->
```

Comment:
Zwave Binding users will not approve this change :-) You do need to add your own binding as a module here.